src/Components/Events/index.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {View , StyleSheet, Text} from 'react-native';
+import EventCard from './Cards';
+import {getevents_highlights} from './../../content/events_and_highlights';
+import SectionSubheader from './../SectionSubheader';
+
+const events_highlight = getevents_highlights();
+
+class Events extends React.Component {
```

Comment:
Use Functional components

Reply:
please tell me what difference it will make as class component are better as for further usage of states